README.md

```diff
@@ -23,9 +23,18 @@ flate2 = "1.0"
 
 ## MSRV (Minimum Supported Rust Version) Policy
 
-This crate supports the current stable and the last stable for the latest version.
-For example, if the current stable is 1.64, this crate supports 1.64 and 1.63.
-Older stables may work, but we don't guarantee these will continue to work.
+This crate supports the current stable and the last stable compiler for the latest version.
+For example, if the current stable is 1.80, this crate supports 1.80 and 1.79.
+
+Older stables may work, but failures may not be treated as a `flate2` bug.
+
+The `Cargo.toml` file specifies a minimum version for which builds passed at some point.
```

Comment:
```suggestion
The `Cargo.toml` file specifies a minimum version for which builds passed at some point using the `rust-version` field.
```

Reply:
Since 'minimum version' is really the `rust-version` field, I wonder if that name should rather be used to reduce the level of abstraction.